var/docs/docker.md

```diff
@@ -50,7 +50,7 @@ To install the fixtures:
 
 ## Developing against the docker
 
-It is possible to mount your source tree and sqlite DB into the container at run time.
+It is possible to mount your source tree and sqlite DB into the container at run time.  **N.B. The sqlite database needs to writable by the www-data user.** Use ```chown 33:33 /path/to/file.sqlite``` on the host machine.
```

Comment:
I'd prefer if the /path/to/file.sqlite example would either match the command example below `chown 33:33 $(pwd)/var/data/kimai.sqlite` or just `chown 33:33 kimai.sqlite`

Reply:
I switched it to just kimai.sqlite as the other path does not match the error generated by Hay777